policysync/ipset.go

```diff
@@ -37,6 +37,8 @@ func newIPSet(update *proto.IPSetUpdate) *ipSetInfo {
 		s.Type = ipsets.IPSetTypeHashIPPort
 	case proto.IPSetUpdate_NET:
 		s.Type = ipsets.IPSetTypeHashNet
+	default:
+		panic("unknown IPSetType")
```

Comment:
You should use `log.WithField("IPSetType", ...).Panic("...")`.  `log.Panic()` makes sure the log gets properly flushed.